airbyte-webapp/src/packages/cloud/locales/en.json

```
@@ -180,6 +180,7 @@
   "freeConnectorProgram.enrollNow": "Enroll now!",
   "freeConnectorProgram.enroll.description": "Enroll in the <b>Free Connector Program</b> to use Alpha and Beta connectors for <b>free</b>.",
   "freeConnectorProgram.enroll.success": "Successfully enrolled in the Free Connector Program",
+  "freeConnectorProgram.enroll.failure": "Unable to verify that payment details were saved successfully",
```

I did wonder if we would like any verbiage about reaching out to support? The user and their browser are faced with a pretty opaque error state. Thoughts, @andyjih?

---

@ambirdsall Good call! Here's a pass: 

```suggestion
  "freeConnectorProgram.enroll.failure": "Unable to verify that payment details were saved successfully. Please contact support for additional help.",
```